### PR TITLE
US043 Find an RU [WIP]

### PR DESCRIPTION
**Pull request by tej99**
As an:
internal User
I need:
to be able to 'find' a Reporting Unit (RU Ref for Business Surveys)
So that:
I can easily locate the RU to deal with any issues

### CODE REVIEW

**Comment by tej99** [comment]
[US043: INT Find an RU [Large]](https://trello.com/c/WyRWFKXq/38-us043-int-find-an-ru-large)

File: docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
       POSTGRES_PASSWORD: postgres
     image: postgres:9.6
     ports:
-      - "5432:5432"
+      - "6432:5432"
```
Comment (andrewmil) [inline docker-compose.yml:10]: Is this an intentional change?

**Review by andrewmil — APPROVED** [review]
:+1: Quality PR

File: docker-compose.yml (hunk 1 above)
Comment (tej99) [reply docker-compose.yml:10]: Yes need it to be 6432 as thats the default database port config.

**Review by gemmai95 — APPROVED** [review]

**Pull request merged**